src/app/activity/activity.ts

```diff
@@ -1,9 +1,19 @@
+import {firestore} from 'firebase/app';
+import Timestamp = firestore.Timestamp;
+
 export class Activity {
-  user: string;
-  user_nickname: string;
-  date: Date;
-  individual: string;
+  action?: string;
+  activity_date: Timestamp;
+  followers: [string];
   individual_id: string;
-  text: string;
-  name: string;
+  individual_name: string;
+  observation_id?: string;
+  phenophase: string;
+  phenophase_name: string;
+  source: string;
+  species: string;
+  species_name: string;
+  type: string;
```

Comment:
Are there a lot of types? I suggest adding allowed strings similar to `type` on `Individual`

Reply:
good point. There we are only handling observations at the moment. In future we might add activities for individual creation e.g.